src/server.ts

```diff
@@ -32,6 +33,331 @@ app.get("/health-check", async (req, res) => {
   }
 });
 
+//-----------------------------------------------------------------------------------------------------requests to DATABASE for USERS table
+//-----------------------------------------------------------------------------------------------------GET request to DATABASE for all users
+app.get("/users", async (req, res) => {
+  const users = await client.query("SELECT * FROM users ORDER BY user_id");
+  res.status(200).json(users.rows);
+});
+//-----------------------------------------------------------------------------------------------------GET request to DATABASE for user by user_id
+app.get<{ user_id: number }>("/users/:user_id", async (req, res) => {
+  const user_id = req.params.user_id;
+
+  if (user_id === undefined) {
```

Comment:
Seems good to consider and prevent possible errors passing through to the database with these undefined checks. 

Maybe could have covered more bases with

```
try{
...
} catch(error) {
console.error(...)
}
```